re_pages/Packages.res

```diff
@@ -450,14 +450,23 @@ let default = (props: props) => {
   | Filtered(pattern) => Resource.applySearch(allResources, pattern)
   }
 
+  let router = Next.Router.useRouter()
+
+  let updateQuery = value =>
+    router->Next.Router.pushObj({
+      pathname: router.pathname,
+      query: value === "" ? Js.Dict.empty() : Js.Dict.fromArray([("searchterm", value)]),
+    })
+
   let onValueChange = value => {
+    updateQuery(value)
```

Comment:
Not sure if it's a good idea to launch the side-effect on callback handles.. might be better to do the dirty work in the `useEffect` whenever the search state changes.